helpers/filters.js

```diff
@@ -67,7 +67,7 @@ exports.check_path = function(path, req, res) {
         return true
     }
 
-    re = new RegExp(/((^etc\/rules\/|^etc\/decoders\/)[A-Za-z_\-\/]+\.{1}xml|(^etc\/lists\/)[A-Za-z_\-\.\/]+)/)
+    re = new RegExp(/((^etc\/rules\/|^etc\/decoders\/)[\w_\-\/]+\.{1}xml|(^etc\/lists\/)[|w_\-\.\/]+)/)
```

Comment:
There is an error here: |w is obviously \w. The '_' is already matched by \w so it can be deleted safely.

Reply:
fixed!